aspnetcore/blazor/hosting-models.md

```diff
@@ -121,29 +119,69 @@ For more information on Microsoft native client frameworks, see the following re
 * [Windows Presentation Foundation (WPF)](/dotnet/desktop/wpf/overview/)
 * [Windows Forms](/dotnet/desktop/winforms/overview/)
 
-:::moniker-end
-
 ## Which Blazor hosting model should I choose?
 
-Select the Blazor hosting model for app development based on the desired features and specifications for the app. The following table shows the primary considerations for selecting the hosting model to help you decide which one you should use.
+[!INCLUDE[](~/blazor/includes/blazor-hybrid-preview-notice.md)]
+
+Select the Blazor hosting model based on the app's feature requirements. The following table shows the primary considerations for selecting the hosting model.
 
 | Feature | Blazor Server | Blazor WebAssembly (WASM) | Blazor Hybrid |
 | --- | :---: | :---: | :---: |
-| Complete .NET API compatibility                | ✔️ | ❌ | ✔️ |
-| Direct access to server sources                | ✔️ | ❌ | ❌ |
-| Small payload size with fast initial load time | ✔️ | ❌ | ✔️ |
-| App code secure and private on the server      | ✔️ | ❌&dagger; | ❌ |
-| Run apps offline once downloaded               | ❌ | ✔️ | ✔️ |
-| Static site hosting                            | ❌ | ✔️ | ❌ |
-| Offloads processing to clients                 | ❌ | ✔️ | ✔️ |
-| Full access to native client capabilities      | ❌ | ❌ | ✔️ |
+| [Complete .NET API compatibility](#complete-net-api-compatibility) | ✔️ | ❌ | ✔️ |
+| [Direct access to server and network resources](#direct-access-to-server-and-network-resources) | ✔️ | ❌&dagger; | ❌&dagger; |
+| [Small payload size with fast initial load time](#small-payload-size-with-fast-initial-load-time) | ✔️ | ❌ | ✔️ |
```

Comment:
I think only Blazor Server has a small payload size and fast load time. Blazor Hybrid apps are native client apps that have to be downloaded and installed.

Reply:
Ah! I see why ... your edit to the original PR ...

https://github.com/dotnet/AspNetCore.Docs/pull/24965/commits/4ddaf34dd78d126d805d0c1ebce9287aa2601c0f

Line 133.

Reply:
Whoops! Sorry! 😳